### PR TITLE
Include cosmos db in persistence selection

### PR DESCRIPTION
**Pull request by yvesgoeleven**

### CODE REVIEW

File: persistence/selecting.md [hunk 1]
```diff
@@ -58,9 +60,11 @@ The non-durable persister is not mentioned in the decision chart, as it is appro
...
+There are several options available when endpoints are hosted in Microsoft Azure. The most commonly used persisters are Azure Storage, Azure CosmosDB and Azure SQL, the fully managed SQL Server solution in Azure.
 
-One factor in the decision is whether the system is fully platform-as-a-service-enabled and whether it is designed to run fully on Azure (in which case, the Azure Table persistence may be appropriate). Alternatively, some organizations are more comfortable managing SQL Server and may choose an infrastructure-as-a-service solution (using the SQL persister with Azure SQL).
+One factor in the decision is whether the system is fully platform-as-a-service-enabled and whether it is designed to run fully on Azure in which case, the Azure Storage Table or Azure CosmosDB persistence may be appropriate. The choice between Azure Storage Table and Azure CosmosDB boils down to a difference in cost vs capabilities.
+
+Alternatively, some organizations are more comfortable managing SQL Server and may choose an infrastructure-as-a-service solution (using the SQL persister with Azure SQL).
```
Comment (danielmarbach) [inline persistence/selecting.md:67]: I know this sentence was already there but frankly, I find it confusing. Above we say

> Azure SQL, the fully managed SQL Server solution in Azure

and then here we say

> and may choose an infrastructure-as-a-service solution (using the SQL persister with Azure SQL).

File: persistence/selecting.md [hunk 2]
```diff
@@ -74,6 +78,15 @@ One factor in the decision is whether the system is fully platform-as-a-service-
 - Automatically scales
 - A turn-key solution, meaning no maintenance
 
+**Azure CosmosDB (preview)**
+
+- Supports local transactions
```
Comment (danielmarbach) [inline persistence/selecting.md:83]:
```suggestion
- Supports transactions on the same partition
```

File: persistence/selecting.md (hunk 1 above)
Comment (yvesgoeleven) [reply persistence/selecting.md:67]: perhaps we can even add 'Sql Server hosted on VM'?

Comment (danielmarbach) [reply persistence/selecting.md:67]: I think it would be more precise and less confusing

Comment (yvesgoeleven) [reply persistence/selecting.md]: done

**Review by danielmarbach — APPROVED** [review]

**Review by HEskandari — APPROVED** [review]

**Pull request merged**